fenix/app/src/main/res/values/strings.xml

```diff
@@ -2235,9 +2235,9 @@
     <!-- Flag showing that the review quality check feature is work in progress. -->
     <string name="review_quality_check_beta_flag" tools:ignore="UnusedResources">Beta</string>
     <!-- Content description (not visible, for screen readers etc.) for opening browser menu button to open review quality check bottom sheet. -->
-    <string name="browser_menu_review_quality_check">Open Review quality check</string>
+    <string name="browser_menu_review_quality_check_1">Open review checker</string>
     <!-- Content description (not visible, for screen readers etc.) for closing browser menu button to open review quality check bottom sheet. -->
-    <string name="browser_menu_review_quality_check_close">Close review quality check</string>
+    <string name="browser_menu_review_quality_check_close_1">Close review checker</string>    
```

Comment:
Might want to remove the trailing spaces, otherwise someone else's editor might do it next time, and create an unnecessary diff.